app/src/main/java/com/example/erabook/firebaseActivities/RegisterSuccessfulDialogFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.erabook.firebaseActivities
+
+import android.app.Dialog
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.erabook.databinding.RegisterSuccessfulBinding
+
+class RegisterSuccessfulDialogFragment : DialogFragment() {
+    private lateinit var binding: RegisterSuccessfulBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = RegisterSuccessfulBinding
+            .inflate(
+                layoutInflater,
+                container, false
+            )
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.apply {
+            OK.setOnClickListener {
+                dialog?.dismiss()
+            }
+            goToLogin.setOnClickListener {
```

Comment:
If goToLogin is a button then add suffix Button at the end please like: goToLoginButton.